core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -135,6 +137,51 @@ def validate_funding_profiles_funding_source(
         ]
 
 
+def validate_sign_off(workbook: dict[str, pd.DataFrame]) -> list["TownsFundRoundFourValidationFailure"] | None:
+    """Validates Name, Role, and Date for the Review & Sign-Off Section
+
+    :param workbook: A dictionary where keys are sheet names and values are pandas
+                     DataFrames representing each sheet in the Round 4 submission.
+    :return: ValidationErrors
+    """
+    sheet = workbook.get("8 - Review & Sign-Off")
+    sheet.replace(r"", np.nan, inplace=True)
+
+    section_151_text_cells = [6, 7, 9]
+    town_board_chair_cells = [13, 14, 16]
+
+    failures = []
+
+    for y_axis in section_151_text_cells:
+        if pd.isnull(sheet.iloc[y_axis, 2]):
+            failures.append(
+                TownsFundRoundFourValidationFailure(
+                    tab="Review & Sign-Off",
+                    section="Section 151 Officer / Chief Finance Officer",
+                    message=(
+                        "You must fill out the " + str(sheet.iloc[y_axis, 1]) + " for this section. "
+                        "You need to get sign off from an S151 Officer or Chief Finance Officer"
+                    ),
+                )
+            )
+
+    for y_axis in town_board_chair_cells:
+        if pd.isnull(sheet.iloc[y_axis, 2]):
+            failures.append(
+                TownsFundRoundFourValidationFailure(
+                    tab="Review & Sign-Off",
+                    section="Town Board Chair",
+                    message=(
+                        "You must fill out the " + str(sheet.iloc[y_axis, 1]) + " for this section. "
+                        "You need to get sign off from a programme SRO"
+                    ),
```

Comment:
Can an fstring be used instead of string concatenation

Reply:
Yes